docs/source/sysadmin_guide/off_chain_settings.rst

```diff
@@ -0,0 +1,300 @@
+
+   #. Locate the ``bind`` settings. If necessary, change these values for your
+      system. The default values are:
+
+        .. code-block:: ini
+
+           bind = [
+             "network:tcp://127.0.0.1:8800",
+             "component:tcp://127.0.0.1:4004",
+             "consensus:tcp://127.0.0.1:5050"
+           ]
+
+      - ``network`` specifies where the validator listens for communication from
+        validators on other nodes
+
+      - ``component`` specifies where the validator listens for the REST API
+
+      - ``consensus`` specifies the settings for the consensus proxy
+
+      .. tip::
+
+         If the bind interface doesn't exist, you might see a ZMQ error in the
+         ``sawtooth-validator`` systemd logs when attempting to start the
+         validator, as in this example:
+
+         .. code-block:: console
+
+            Jun 02 14:50:37 ubuntu validator[15461]:   File "/usr/lib/python3.5/threading.py", line 862, in run
+            ...
+            Jun 02 14:50:37 ubuntu validator[15461]:   File "zmq/backend/cython/socket.pyx", line 487, in zmq.backend.cython.socket.Socket.bind (zmq/backend/cython/socket.c:5156)
+            Jun 02 14:50:37 ubuntu validator[15461]:   File "zmq/backend/cython/checkrc.pxd", line 25, in zmq.backend.cython.checkrc._check_rc (zmq/backend/cython/socket.c:7535)
+            Jun 02 14:50:37 ubuntu validator[15461]: zmq.error.ZMQError: No such device
+            Jun 02 14:50:37 ubuntu systemd[1]: sawtooth-validator.service: Main process exited, code=exited, status=1/FAILURE
+            Jun 02 14:50:37 ubuntu systemd[1]: sawtooth-validator.service: Unit entered failed state.
+            Jun 02 14:50:37 ubuntu systemd[1]: sawtooth-validator.service: Failed with result 'exit-code'.
+
+#. Set the peering type and peer list (directly connected nodes) for this
+   Sawtooth node.
+
+   a. Locate the ``peering`` setting, which specifies the type of peering
+      approach the validator should take: static or dynamic. Change this setting
+      to ``dynamic`` unless static peering is required. For more information,
+      see :doc:`configuring_sawtooth/validator_configuration_file`.
+
+      .. code-block:: ini
+
+         peering = "dynamic"
+
+   #. Find the ``peers`` setting and enter the URLs for other validators on the
+      network.
+
+      * If ``peering`` is  ``dynamic``, you can enter a partial list of URLs.
+        Sawtooth will automatically discover the other nodes on the network.
+
+      * If ``peering`` is  ``static``, you must list the URLs of **all** peers
+        that this node should connect to.
+
+      Use the format ``tcp://{hostname}:{port}`` for each peer. Specify multiple
+      peers in a comma-separated list. For example:
+
+      .. code-block:: ini
+
+         peers = ["tcp://node1:8800", "tcp://node2:8800", "tcp://node3:8800"]
+
+   #. Find the ``seeds`` setting, which specifies the peers to use for the
+      initial connection to the validator network.
```

Comment:
`--seeds` is only useful in `dynamic` peering mode, and it specifies the other validators that this validator will first request peering information from in order to peer dynamically.